source/code-generation/query-converter.txt

```diff
@@ -23,3 +23,12 @@ preview. To request private preview access, use the sign-up form within the
 Relational Migrator application:
 
 .. include:: /includes/fact-request-private-preview.rst
+
+.. toctree::
+   :hidden:
+   :titlesonly:
+
+   /code-generation/query-converter/enable-query-converter
+   /code-generation/query-converter/convert-queries
+   /code-generation/query-converter/convert-stored-procedures
```

Comment:
Before you can convert stored procedures or views, you need to import them into the project. I think there's an argument for having a dedicated procedure for importing/managing DB objects and referencing them from the conversion topics.

Reply:
Agreed, I made a comment under https://jira.mongodb.org/browse/DOCSP-33003, detailing this and will plan to add one more task to the query converter content outlining the procedure.

Reply:
Side note, if you updated the button on the Mapping tab, `manage` to `refresh objects` we could reuse the content as a general task and include it else where. Right now the 2 workflows do the same thing but the buttons are named slightly different. 

I.E.:

Mapping > Manage > `Refresh Schema`
Code Generation > Query Converter > Manage database objects > `Refresh Objects`